dmcontent/questions.py

```diff
@@ -41,10 +41,26 @@ def filter(self: TQuestion, context, dynamic=True, inplace_allowed: bool = False
 
     def _should_be_shown(self, context):
         return all(
-            depends["on"] in context and (context[depends["on"]] in depends["being"])
+            self._is_dependency_valid(depends["on"], depends["being"], context)
             for depends in self.get("depends", [])
         )
 
+    def _is_dependency_valid(self, key, condition, context):
+        is_valid = False
+        if key in context:
+            if context[key] in condition:
+                is_valid = True
+            else:
+                boolean_array = []
+                for cond in condition:
```

Comment:
If there are potentially multiple conditions, why are we using the singular `condition`?